src/engine/SCons/Node/Python.py

```diff
@@ -173,6 +176,23 @@ def get_csig(self, calc=None):
         self.get_ninfo().csig = contents
         return contents
 
+
+def ValueWithMemo(value, built_value=None):
+    global _memoLookupMap
+
+    # No current support for memoizing a value that needs to be built.
+    if built_value:
```

Comment:
Can you add a test to cover this as well?

Reply:
Done, thanks!